dropshot/src/config.rs

```diff
@@ -113,9 +116,108 @@ impl Default for ConfigDropshot {
     fn default() -> Self {
         ConfigDropshot {
             bind_address: "127.0.0.1:0".parse().unwrap(),
-            request_body_max_bytes: 1024,
+            default_request_body_max_bytes: 1024,
             default_handler_task_mode: HandlerTaskMode::Detached,
             log_headers: Default::default(),
         }
     }
 }
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default)]
+struct DeserializedConfigDropshot {
+    bind_address: SocketAddr,
+    default_request_body_max_bytes: usize,
+    // Previous name for default_request_body_max_bytes. Present only to guide
+    // users to the new name.
```

Comment:
worth noting the version when this change was made? Might save us time in the future if/when we choose to EOL this migration guidance.

Reply:
Done.